slurm/make_ref.jl

```diff
@@ -8,29 +8,14 @@ using OPFGenerator
 config_file = ARGS[1]
 config = TOML.parsefile(config_file)
 casename = config["ref"]
-export_dir = config["export_dir"]
+export_dir = pop!(config, "export_dir")
+pop!(config, "slurm")
```

Comment:
Out of curiosity: why do you `pop!` these?

Reply:
So they don't show up in the JSON file since they are not relevant to the dataset itself